smart-contract/assembly/__tests__/FileBuilder.ts

```diff
@@ -127,6 +134,18 @@ class FileBuilder {
     return this;
   }
 
+  deleteFile(file: ChunkDelete): void {
+    deleteFile(
+      new Args().addSerializableObjectArray<ChunkDelete>([file]).serialize(),
+    );
+  }
```

Comment:
I would remove deleteFile function especially because in the end it will be the same as deleteFiles